service.js

```diff
@@ -9,7 +9,7 @@ var request = require('request-promise');
 const bunyan = require("bunyan");
 
 class HealthCheckService {
-    constructor(services) {
+    constructor(ylpVersion, services) {
```

Comment:
You will need to update the tests to use this new signature, OR to switch both variables